docs/build-on-linea/linea-version/index.mdx

```diff
@@ -11,6 +11,47 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 
 Find out all you need to know about the latest Linea versions.
 
+## Alpha v3 release notes
+
+### Summary
+
+The main objective of Alpha v3 is to implement EIP-4844 on Linea, following its introduction to 
+Ethereum mainnet with the Dencun upgrade on March 13.
+
+### Features
+
+#### March 26: EIP-4844
+
+Begin using blob data to post data to L1, significantly reducing data availability costs. 
+Blob data replaces `calldata` and is significantly cheaper.
+
+Multiple contracts have been updated and support blob data. Notable new contract addresses include:
+
+- New verifier contract
+  - Linea Sepolia: [0x5ca5dBf7Cb8F3f3c92E04B16FF5fCA1cdf147f79](https://sepolia.lineascan.build/address/0x5ca5dBf7Cb8F3f3c92E04B16FF5fCA1cdf147f79)
+  - Linea mainnet: TBC
+- New L1 message service
+  - Linea Sepolia: [0xB218f8A4Bc926cF1cA7b3423c154a0D627Bdb7E5](https://sepolia.lineascan.build/address/0xB218f8A4Bc926cF1cA7b3423c154a0D627Bdb7E5)
+  - Linea mainnet: TBC
+- New L2 rollup contract
+  - Linea Sepolia: [0x971e727e956690b9957be6d51Ec16E73AcAC83A7](https://sepolia.lineascan.build/address/0x971e727e956690b9957be6d51Ec16E73AcAC83A7)
+  - Linea mainnet: TBC
+
+All contracts have been audited by OpenZeppelin. A link to their reports will be shared here once
+available.
+
+#### March 27
+
+- Block time reduced to 3 seconds. This change aims to improve user experience by increasing
+the network's tps (transactions per second) capacity.
```

Comment:
This change increases the throughput of the network to avoid any potential L2 execution bottlenecks from increased Linea activity.